modules/signals/rxjs-interop/spec/rx-method.spec.ts

```diff
@@ -231,4 +238,122 @@ describe('rxMethod', () => {
     TestBed.flushEffects();
     expect(counter()).toBe(4);
   });
+
+  it('completes on manual destroy with Signals', () => {
+    TestBed.runInInjectionContext(() => {
+      let completed = false;
+      const counter = signal(1);
+      const fn = rxMethod<number>(finalize(() => (completed = true)));
+      TestBed.flushEffects();
+      fn(counter);
+      fn.unsubscribe();
+      expect(completed).toBe(true);
+    });
+  });
+
+  describe('Signals and effect injector', () => {
+    @Injectable({ providedIn: 'root' })
+    class GlobalService {
+      rxMethodStatus = 'none';
+      log = rxMethod<string>(
+        pipe(
+          tap({
+            next: () => (this.rxMethodStatus = 'started'),
+            finalize: () => (this.rxMethodStatus = 'destroyed'),
+          })
+        )
+      );
+    }
+
+    @Component({
+      selector: `app-storeless`,
+      template: ``,
+      standalone: true,
+    })
+    class WithoutStoreComponent {}
+
+    function setup(WithStoreComponent: new () => unknown): GlobalService {
+      TestBed.configureTestingModule({
+        providers: [
+          provideRouter([
+            { path: 'with-store', component: WithStoreComponent },
+            {
+              path: 'without-store',
+              component: WithoutStoreComponent,
+            },
+          ]),
+          provideLocationMocks(),
+        ],
+      });
+
+      return TestBed.inject(GlobalService);
+    }
+
+    it('should destroy with component injector when rxMethod is in root and RxMethod in component', async () => {
```

Comment:
for consistency with other tests from the signals package:

```suggestion
    it('destroys with component injector when rxMethod is in root and RxMethod in component', async () => {
```

the same convention should be applied to other tests